PBN RCT.Rmd

```diff
@@ -69,35 +69,144 @@ followup_data_deduplicated <- followup_data %>%
+```
+
+```{r clean data}
+#Make necessary variables factors
+###STILL NEED TO DO GENDER AND RACE/ETHNICITY, SEE IF I CAN DOWNLOAD DATA BETTER###
+participants_with_followup$b_dem_saab <- factor(
+participants_with_followup$b_dem_saab,
+  levels = c(0, 1),
+  labels = c("No", "Yes")
+)
+participants_with_followup$b_dem_sexuality <- factor(
+  participants_with_followup$b_dem_sexuality,
+  levels = c(1, 2, 3, 4, 5, 6, 7, 8, 9, 10),
+  labels = c("Straight/Heterosexual", "Gay/Lesbian/Homosexual", "Bisexual", "Pansexual", "Queer", "Asexual", "Other (please specify)", "Unsure/Questioning", "I do not use a label", "I do not want to respond")
+)
+participants_with_followup$b_dem_sex <- factor(
+  participants_with_followup$b_dem_sex,
+  levels = c(1, 2, 3),
+  labels = c("Male", "Female", "Intersex")
+)
+participants_with_followup$b_current_mood <- factor(
+  participants_with_followup$b_current_mood,
+  levels = c(0, 1),
+  labels = c("No", "Yes")
+)
+participants_with_followup$b_past_mood <- factor(
+  participants_with_followup$b_past_mood,
+  levels = c(0, 1),
+  labels = c("No", "Yes")
+)
+participants_with_followup$b_unmet_mood_now <- factor(
+  participants_with_followup$b_unmet_mood_now,
+  levels = c(0, 1),
+  labels = c("No", "Yes")
+)
+participants_with_followup$b_unmet_mood_past <- factor(
+  participants_with_followup$b_unmet_mood_past,
+  levels = c(0, 1),
+  labels = c("No", "Yes")
+)
+participants_with_followup$b_current_ed <- factor(
+  participants_with_followup$b_current_ed,
+  levels = c(0, 1),
+  labels = c("No", "Yes")
+)
+participants_with_followup$b_past_ed <- factor(
+  participants_with_followup$b_past_ed,
+  levels = c(0, 1),
+  labels = c("No", "Yes")
+)
+participants_with_followup$b_unmet_ed_now <- factor(
+  participants_with_followup$b_unmet_ed_now,
+  levels = c(0, 1),
+  labels = c("No", "Yes")
+)
+participants_with_followup$b_unmet_ed_past <- factor(
+  participants_with_followup$b_unmet_ed_past,
+  levels = c(0, 1),
+  labels = c("No", "Yes")
+)
+participants_with_followup$b_hunger_1 <- factor(
+  participants_with_followup$b_hunger_1,
+  levels = c(1, 2, 3),
+  labels = c("Never True", "Sometimes True", "Often True")
+)
+participants_with_followup$b_hunger_2 <- factor(
+  participants_with_followup$b_hunger_2,
+  levels = c(1, 2, 3),
+  labels = c("Never True", "Sometimes True", "Often True")
+)
+participants_with_followup$f_new_support_mood <- factor(
+  participants_with_followup$f_new_support_mood,
+  levels = c(0, 1),
+  labels = c("No", "Yes")
+)
+participants_with_followup$f_new_support_ed <- factor(
+  participants_with_followup$f_new_support_ed,
+  levels = c(0, 1),
+  labels = c("No", "Yes")
+)
 
 #Create a binary variable to identify the treatment group
-participants_with_followp <- participants_with_followp %>%
+participants_with_followup <- participants_with_followup %>%
   mutate(treatment_group = Condition == "Intervention")
+###WILL THIS WORK IN MISSFOREST OR DO WE NEED TO DUMMY CODE###
```

Comment:
For dichotomous 1/0 variables I bet it doesn't make a difference whether it's factor or numeric type?